packages/orchestration/test/examples/stake-ica.contract.test.ts

```diff
@@ -0,0 +1,156 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { setUpZoeForTest } from '@agoric/zoe/tools/setup-zoe.js';
+import { heapVowE as E } from '@agoric/vow/vat.js';
+import path from 'path';
+import { makeNotifierFromSubscriber } from '@agoric/notifier';
+import type { Installation } from '@agoric/zoe/src/zoeService/utils.js';
+import {
+  QueryBalanceRequest,
+  QueryBalanceResponse,
+} from '@agoric/cosmic-proto/cosmos/bank/v1beta1/query.js';
+import { commonSetup } from '../supports.js';
+import { type StakeIcaTerms } from '../../src/examples/stakeIca.contract.js';
+import chainInfo from '../../src/fetched-chain-info.js';
```

Comment:
please name `fetchedChainInfo` to match its module and make more clear the source when reading within `getChainTerms`

Reply:
Rename done and included `fetchChainInfo` as an optional second argument typed as `Record<string, CosmosChainInfo>` - `ChainInfo` was giving a hard time and this is a cosmos-specific contract